docs/azure_arc.md

```diff
@@ -0,0 +1,48 @@
+# Deploying to Azure Arc
+
+This template supports deployment to Azure Arc. The following document lists the requirements to setup Azure Arc to support the template, as well as how you deploy the application to an Arc-enable Kubernetes Cluster.
+
+- [Deploying to Azure Arc](#deploying-to-azure-arc)
+  - [Prerequisites](#prerequisites)
+  - [Regions and resource group support](#regions-and-resource-group-support)
+    - [Azure Arc enabled App Service](#azure-arc-enabled-app-service)
+
+## Prerequisites
```

Comment:
Can we not show them the az cli commands to get this setup?

Can we not provision/configure with bicep?

Reply:
It's a multitude:
1. Create a K8s cluster (anywhere)
2. Arc-enable (connect)
3. Add extensions (data and app)
4. Create custom location

The linked article describes the concepts and the pointers to docs. I'm usin gthis script for an environment on AKS:https://github.com/mikkelhegn/arc. The Arc model has this concept of custom locations, which are mapped to namespaces in a K8 cluster. My assumption is that a platform team will provide the namespace (custom location) for the app team, same way as a subscription would be provided by someone else for Azure (in an Enterprise setup). Scope-wise, I belive this is out for the template.